engine/security/trust/trust_delegation.md

```diff
@@ -5,33 +5,31 @@ title: Delegations for content trust
 ---
 
 Delegations in Docker Content Trust (DCT) allow you to control who can and cannot sign
-an image tag. A delegation will have a pair of delegation keys, public and 
-private. A delegation could contain multiple pairs of keys, contributors, to 
-allow multiple users to be part of a delegation, and to support key rotation.  
+an image tag. A delegation will have a pair of private and public delegation keys. A delegation 
+could contain multiple pairs of keys and contributors in order to a) allow multiple users 
+to be part of a delegation, and b) to support key rotation.  
 
 The most important delegation within Docker Content Trust is `targets/releases`.
 This is seen as the canonical source of a trusted image tag, and without a 
 contributor's key being under this delegation, they will be unable to sign a tag.
 
 Fortunately when using the `$ docker trust` commands, we will automatically 
-initialise a repository, manage the repository keys, and when a collaborator
-gets added with `docker trust signer add` we will add their key to the 
-`targets/releases` delegation automatically. 
+initialize a repository, manage the repository keys, and add a collaborator's key to the 
+`targets/releases` delegation via `docker trust signer add`. 
 
 ## Configuring the Docker Client
 
-By default the `$ docker trust` commands are expecting the Notary server URL
-to be the same as the Docker Registry URL specified in the image tag. When 
-using the Docker Hub or Docker Trusted Registry this is the case as a internal 
-proxy redirects the request; however for self hosted environments or 3rd party 
-registries you will need to specify an alternative URL for the notary server. 
+By default, the `$ docker trust` commands expect the notary server URL
+to be the same as the Docker Trusted Registry (DTR) URL specified in the image tag. This is because an 
```

Comment:
@bermudezmt This is incorrect. The Registry URL is specified in an image tag. Not a DTR URL.

Reworded in https://github.com/docker/docker.github.io/pull/8433